src/pages/projectSettings/tabs/PeriodicBuildsTab/getFormSchema.ts

```diff
@@ -0,0 +1,126 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import widgets from "components/SpruceForm/Widgets";
+import { timeZones } from "constants/fieldMaps";
+import { form, ProjectType } from "../utils";
+
+const { overrideRadioBox } = form;
+
+export const getFormSchema = (
+  projectType: ProjectType,
+  timezone: string
+): {
+  fields: Record<string, Field>;
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  fields: {},
+  schema: {
+    type: "object" as "object",
+    description: "Configure tasks to run at fixed intervals.",
+    ...overrideRadioBox(
+      "periodicBuilds",
+      ["Override Repo Commands", "Default to Repo Commands"],
+      {
+        type: "array" as "array",
+        default: [],
+        items: {
+          type: "object" as "object",
+          properties: {
+            intervalHours: {
+              type: "number" as "number",
+              title: "Interval",
+            },
+            configFile: {
+              type: "string" as "string",
+              title: "Config File",
+              minLength: 1,
+              default: "",
+            },
+            alias: {
+              type: "string" as "string",
+              title: "Alias",
+              default: "",
+            },
+            message: {
+              type: "string" as "string",
+              title: "Message",
+              default: "",
+            },
+            timezone: {
```

Comment:
Does everybody have a default timezone? Or else should we populating this with our default timezone (which I assume is us-east)
<img width="516" alt="image" src="https://user-images.githubusercontent.com/26798134/161625592-2a72c1fd-57af-4e06-9271-879dd92f95a7.png">

Reply:
Users are not required to have a timezone set. In places like the hosts table, if a timezone is not set, we use the user's local time. I can go with that approach here unless you think defaulting to Eastern Time makes more sense.

Reply:
Using the user's local time makes sense, I just think this should show something.